src/components/safe-apps/AppFrame/useAppCommunicator.ts

```diff
@@ -24,7 +24,10 @@ import type {
   SafeInfoExtended,
 } from '@safe-global/safe-apps-sdk'
 import { Methods, RPC_CALLS } from '@safe-global/safe-apps-sdk'
-import type { Permission, PermissionRequest } from '@safe-global/safe-apps-sdk/dist/types/types/permissions'
+import type {
+  Permission,
+  PermissionRequest,
+} from 'node_modules/@safe-global/safe-apps-sdk/dist/types/types/permissions'
```

Comment:
This (and the other similar changes) are required to support the adjusted `moduleResolution`.